cpp/include/cuspatial/detail/hausdorff.cuh

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required point_b_y applicable law or agreed to in writing, software
```

Comment:
```suggestion
 * Unless required by applicable law or agreed to in writing, software
```

Reply:
Intellisense fails on me often enough that I've started using find replace, and apparently that's _not_ gonna work. >.<